pil-analyzer/src/evaluator.rs

```diff
@@ -1355,10 +1321,10 @@ mod test {
         .to_string()
     }
 
-    pub fn evaluate_function<T: FieldElement>(input: &str, function: &str) -> T {
+    pub fn evaluate_function<T: FieldElement>(input: &str, poly_fn: &PolynomialReference) -> T {
```

Comment:
I think this can keep its signature (it's a helper function) or do you need the functionality to call type-arg-qualified names?